src/scripts/main.js

```diff
@@ -1 +1,51 @@
 'use strict';
+
+const clicks = ['click', 'contextmenu'];
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error(`First promise was rejected`)), 3000);
```

Comment:
The `firstPromise` is correctly set to reject after 3 seconds if not clicked, which aligns with the task requirements.